packages/core/src/controllers/OnRampController.ts

```diff
@@ -16,6 +16,7 @@ export interface OnRampControllerState {
   providers: OnRampProvider[]
   selectedProvider: OnRampProvider | null
   error: string | null
+  enabled: boolean
```

Comment:
Should this perhaps live in `OptionsController` instead? (to keep all feature flags in one place)